draft-ietf-quic-http.md

```diff
@@ -1150,9 +1156,8 @@ error of type H3_CLOSED_CRITICAL_STREAM.  Connection errors are described in
 
 A pair of unidirectional streams is used rather than a single bidirectional
 stream.  This allows either peer to send data as soon as it is able.  Depending
-on whether 0-RTT is enabled on the QUIC connection, either client or server
-might be able to send stream data first after the cryptographic handshake
-completes.
```

Comment:
With 0-RTT the client can send data before the cryptographic handshake completes.